Source/WEB_ROOT/Inventory/js/assets.js

```diff
@@ -13,22 +13,27 @@ $j(document).ready(function(){
     $j('#searchBoxBtn').click(function(){
 	    //The Search button was clicked
 		//console.log("Search was clicked : " + $j('#searchBox').val());
-		MakeRequest($j('#searchBox').val());
-
+		if($j('#searchBox').val() == "")

```

Comment:
The only little nice to have here would be to look for min values (don't search until I have three letters, and I don't have an empty string. Something like "  ".  A nice little implementation on stack overflow. https://stackoverflow.com/questions/22849666/check-if-string-is-empty-with-jquery